test/Microsoft.IdentityModel.Tokens.Saml.Tests/Saml2SecurityTokenHandler.Extensibility.Lifetime.cs

```diff
@@ -0,0 +1,291 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Same comments in other test class apply here.

Reply:
ack